Common/OpenCL/ITKimprovements/itkGPUDataManager.cxx

```diff
@@ -155,7 +155,9 @@ GPUDataManager::UpdateCPUBuffer()
     return;
   }
 
+#ifndef __wasi__
   MutexHolderType holder(m_Mutex);
+#endif
```

Comment:
Does WASI support the OpenCL elastix components in general?

Reply:
OpenCL features are not supported.

Reply:
OK, thanks, so then source files from Common/OpenCL do not need to be adjusted, right? Because they are excluded from compilation anyway, when doing a WASI build.

Reply:
OK, I will remove this from the patch.